scripts/add_electricity.py

```diff
@@ -68,6 +70,15 @@ def load_costs():
     return costs
 
 
+def load_costs_ts():
+
+    fuel_costs = pd.read_csv(snakemake.input.fuel_cost_timeseries,
+                             parse_dates=True,
+                             index_col=['report_date'])
+
+    return fuel_costs
+
+
 def load_existing_generators():
```

Comment:
"Exports," here, just means that the total "demand" is equal to the total historical generation (since IL is an energy exporter). Removing "exports" means the total demand is equal to the historical retail sales within IL (about 2/3 of total generation).